dp/core/worker.c

```diff
@@ -633,6 +644,10 @@ void do_work(void)
 
     printf("Worker %d started with tid %d\n", cpu_nr_, worker_tid);
 
+    printf("concord_preempt_now: %p\n", &concord_preempt_now);
+    cpu_preempt_points[cpu_nr_] = &concord_preempt_now;
+
```

Comment:
Lose one line

Reply:
updated

Reply:
No no. I meant the empty lines after :)
Keep the print

Reply:
:) both print and empty lines removed, since the print used only for debug to see the thread local pointer address, it's not necessary.